js/modules/k6/http/http.go

```diff
@@ -146,3 +149,27 @@ func (*HTTP) setRequestCookies(req *http.Request, jar *cookiejar.Jar, reqCookies
 		}
 	}
 }
+
+func (*HTTP) debugRequest(state *common.State, req *http.Request, description string) {
+	if state.Options.HttpDebug.String != "" {
+		dump, err := httputil.DumpRequestOut(req, state.Options.HttpDebug.String == "full")
+		if err != nil {
+			log.Fatal(err)
+		}
+		logDump(description, dump)
+	}
+}
+
+func (*HTTP) debugResponse(state *common.State, res *http.Response, description string) {
+	if state.Options.HttpDebug.String != "" {
+		dump, err := httputil.DumpResponse(res, state.Options.HttpDebug.String == "full")
```

Comment:
We should make sure `res` is not `nil` before calling `httputil.DumpResponse`, I got a segfault because `httputil.DumpResponse` tried to access `res.Body` when testing this PR just now:

```
...
net/http/httputil.DumpResponse(0x0, 0x1801300, 0xc42e3e1c01, 0xc430143380, 0xc42e3e1cb8, 0xbe9488be22589787, 0x15b59b8312)
	/usr/local/Cellar/go/1.9.3/libexec/src/net/http/httputil/dump.go:284 +0x51
github.com/loadimpact/k6/js/modules/k6/http.(*HTTP).debugResponse(0xc420062000, 0xc42fd481e0, 0x0, 0x18559e9, 0x8)
	/Users/robin/Projects/go/src/github.com/loadimpact/k6/js/modules/k6/http/http.go:165 +0x66
github.com/loadimpact/k6/js/modules/k6/http.(*HTTP).request(0xc420062000, 0x1d36560, 0xc42fe0c4b0, 0xc4220a9c80, 0xc42fd481e0, 0x1851232, 0x3, 0xc430143280, 0xc422cd19a2, 0x2a, ...)
	/Users/robin/Projects/go/src/github.com/loadimpact/k6/js/modules/k6/http/http_request.go:279 +0x9e6
github.com/loadimpact/k6/js/modules/k6/http.(*HTTP).Request(0xc420062000, 0x1d36560, 0xc42fe0c4b0, 0x1851232, 0x3, 0x1d3b300, 0xc420ca1d20, 0xc42e3e25f8, 0x1, 0x1, ...)
	/Users/robin/Projects/go/src/github.com/loadimpact/k6/js/modules/k6/http/http_request.go:87 +0x208
github.com/loadimpact/k6/js/modules/k6/http.(*HTTP).Get(0xc420062000, 0x1d36560, 0xc42fe0c4b0, 0x1d3b300, 0xc420ca1d20, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/robin/Projects/go/src/github.com/loadimpact/k6/js/modules/k6/http/http_request.go:50 +0x175
...
```

Reply:
Done.